extensions/amp-story/1.0/amp-story-interactive-poll.css

```
@@ -125,5 +125,65 @@
 }
 
 .i-amphtml-story-interactive-poll-container:not(.i-amphtml-story-interactive-has-data) .i-amphtml-story-interactive-option-percentage {
-  display: none !important;
+  visibility: hidden !important;
+}
+
+.i-amphtml-story-interactive-poll-container.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection) 
```

```suggestion
.i-amphtml-story-interactive-active:not(.i-amphtml-story-interactive-post-selection) 
```

I think we can remove this first selector, right?